### PR TITLE
add fspot and xspot for starspots as control parameters

### PR DESCRIPTION
**Pull request by mjoyceGR**

### CODE REVIEW

**Review by pmocz — APPROVED** [review]
🚀🚀🚀

File: star/defaults/controls.defaults [hunk 1]
```diff
@@ -4636,6 +4636,26 @@
...
+      ! * SPOTF (hereafter fspot): a coverage fraction, or "spot filling factor" (in the notation of the YREC documentation); and
+
+      ! * SPOTX (hereafter xspot): the temperature contrast between the spotted and unspotted regions: xspot = T_spot/T_photosphere.
+
+      ! The coverage fraction is set to fspot = 0.34 
+      ! (for consistency with observations of low-mass stars: 
+      ! Cao et al., 2022 https://ui.adsabs.harvard.edu/abs/2022ApJ...924...84C
+      ! and the temperature contrast is set to xspot = 0.85 (also from fits to observations).
+
+    fspot = 1.0
```
Comment (evbauer) [inline star/defaults/controls.defaults:4656]: > Heads up that the docs for the defaults won't render quite right here. (See https://docs.mesastar.org/en/latest/reference/controls.html#star-spot-controls)
> 
> See `$MESA_DIR/star/defaults/FORMAT` for the constraints on how these should be formatted. In particular, this should look something like:
> 
> ```
>       ! fspot
>       ! ~~~~~
> 
>       ! ::
> 
>    fspot = 1.0
> ```
> 
> ...and similar for `xspot`.

Comment (pmocz) [reply star/defaults/controls.defaults:4656]: Great catch! I can help fix this

Comment (evbauer) [reply star/defaults/controls.defaults:4656]: Thanks!

**Comment by mjoyceGR** [comment]
whoops, also need to move a few more things...recalling the PR and will get back to this tomorrow

**Pull request merged**